src/EnergyPlus/Material.hh

```diff
@@ -561,49 +565,123 @@ namespace Material {
         Real64 InitialPressure = 0.0;     // Window(s) pressure in time of fabrication
     };
 
-    struct ScreenProperties
+    int constexpr maxMixGases = 5;
+
+    struct GasCoeffs
     {
-        // Members
-        int MaterialNumber = 0; // Material pointer for the screen
-        Real64 BmBmTrans = 0.0; // Beam solar transmittance (dependent on sun angle)
+        Real64 c0 = 0.0;
+        Real64 c1 = 0.0;
+        Real64 c2 = 0.0;
+    };
+
+    struct Gas {
+        GasType type = GasType::Custom;
+        GasCoeffs con = GasCoeffs();
+        GasCoeffs vis = GasCoeffs();
+        GasCoeffs cp = GasCoeffs();
+        Real64 wght = 0.0;
+        Real64 specHeatRatio = 0.0;
+    };
+        
+    extern const std::array<Gas, 10> gases;
+
+    struct MaterialGasMix : public MaterialBase {
+        //  up to 5 gases in a mixture [Window gas only].  It is defined as parameter (GasCoefs)
+        int numGases = 0;                             // Number of gases in a window gas mixture
+
+        std::array<Real64, maxMixGases> gasFracts = {0.0};
+        std::array<Gas, maxMixGases> gases = {Gas()};
+            
+        GapVentType gapVentType = GapVentType::Sealed;               // Gap Ven type for equivalent Layer window model
+            
+        virtual bool dummy() { return true; }
```

Comment:
Can probably get rid of this if I add a virtual destructor. I should just do that.